tests/Shared/XUnitLoggerTests.cs

```diff
@@ -659,6 +659,41 @@ public static void XUnitLogger_Log_Logs_Message_If_Scopes_Included_And_There_Is_
         outputHelper.Received(1).WriteLine(expected);
     }
 
+#if NET8_0_OR_GREATER
+
+    [Fact]
+    public static void XUnitLogger_Log_Logs_Message_With_TimeProvider()
+    {
+        // Arrange
+        var outputHelper = Substitute.For<ITestOutputHelper>();
+        string name = "MyName";
+
+        var timeProvider = Substitute.For<TimeProvider>();
+        timeProvider.LocalTimeZone.Returns(TimeZoneInfo.Utc);
+        timeProvider.GetUtcNow().Returns(new DateTimeOffset(2019, 05, 12, 13, 47, 41, 123, 456, TimeSpan.Zero));
```

Comment:
Shouldn't this be setting up `GetLocalNow()` as that's what's being used in the logger? If it's because `GetLocalNow()` is implemented in terms of `GetUtcNow()`, then we should just use that in the logger instead as then it's less to set up.

Reply:
Sorry yes you're right, it's been a while since I've looked at the code.

Local time is used, but then the default is to format it in UTC as with `"u"` as the format string, but that can be customised to give a local result if desired.

Based on that, then using `GetLocalNow()` is correct 😅